### PR TITLE
sstriegs-typography

### PR DESCRIPTION
**Pull request by sstriegs**
Summary:
- using `vw` to make h1s more responsive
- larger loading text
- record page visual tweaks
- lined up tap targets on mobile
- moar hover states

### CODE REVIEW

**Review by dylanjha — APPROVED** [review]
@sstriegs 🔥

**Comment by dylanjha** [comment]
Typescript compiler is not happy, I'll fix that up 🔧

File: pages/v/[id].tsx [hunk 1]
```diff
@@ -100,6 +100,10 @@ const Playback: React.FC<Props> = ({ playbackId, shareUrl, poster }) => {
         .error-message {
           color: #ccc;
         }
+        .loading {
+          font-size: 8vw;
+          max-width: 100%;
+        }
```
Comment (dylanjha) [inline pages/v/[id].tsx:106]: I can see that `FullpageLoader` already has these styles `font-size: 8vw; max-width: 100%;` applied to it within the component, so I'm going to remove the `className` and these styles here (that's what Typescript was complaining about)

Comment (sstriegs) [reply pages/v/[id].tsx]: Ahh I see, yeah something weird seemed to be happening with the "Loading player" text and I thought this fixed it but maybe it just complicated it more

**Pull request merged**